tests/engine/test_ports.py

```diff
@@ -119,3 +119,26 @@ def test_lambda_default(self):
         inputs = port_namespace.pre_process({'port': Int(3)})
         assert isinstance(inputs['port'], Int)
         assert inputs['port'].value == 3
+
+    @pytest.mark.parametrize('is_metadata', (True, False))
+    def test_serializer(self, is_metadata):
```

Comment:
I think it would be nice to have a more high-level test as well. Perhaps you can simply modify some existing CalcJob / WorkChain tests to exercise the new default serializer?

Reply:
Added a test